src/model/Car.js

```diff
@@ -0,0 +1,21 @@
+export class Car {
+  #name;
+  #count;
+
+  constructor(name) {
+    this.#name = name;
+    this.#count = 0;
+  }
+
+  getName() {
+    return this.#name;
+  }
+
+  getCount() {
+    return this.#count;
+  }
```

Comment:
getName()과 getCount()는 단순히 name과 count를 반환하는 역할로 보이는데 접근자 프로퍼티를 사용해서 속성에 접근하는 것도 괜찮을 것 같아요 !

```suggestion
  get name() {
    return this.#name;
  }

  get count() {
    return this.#count;
  }
```

Reply:
좋은 접근법인거 같아요!! 한번 찾아보겠습니다..!